RELEASE/scripts/autoscend/paths/dark_gyffte.ash

```diff
@@ -363,16 +363,38 @@ boolean bat_shouldPickSkills(int hpLeft)
 	return false;
 }
 
+boolean bat_haveEnsorcelee() // checks if you have a current Ensorceled Monster
+{
+	if(my_class() != $class[Vampyre] || !auto_have_skill($skill[Ensorcel]))
+		return false;
+
+	if(get_property("ensorcelee") == $monster[none])
+		return false;
+	
+	return true;
```

Comment:
could probably replace this entire function with
`return get_property("ensorcelee") != $monster[none]`

Reply:
```suggestion
	return get_property("ensorcelee") != "";
```